app/services/support_interface/external_report_candidates_export.rb

```diff
@@ -0,0 +1,268 @@
+                END age_group,
+                CASE
+                  WHEN 'recruited' = ANY(ARRAY_AGG(ac.status)) THEN 'Recruited'
+                  WHEN 'offer_deferred' = ANY(ARRAY_AGG(ac.status)) THEN 'offer_deferred'
+                  WHEN 'pending_conditions' = ANY(ARRAY_AGG(ac.status)) THEN 'Conditions pending'
+                  WHEN 'interviewing' = ANY(ARRAY_AGG(ac.status)) THEN 'Received an offer'
+                  WHEN 'offer' = ANY(ARRAY_AGG(ac.status)) THEN 'Received an offer'
+                  WHEN 'awaiting_provider_decision' = ANY(ARRAY_AGG(ac.status)) THEN 'Awaiting provider decisions'
+                  WHEN 'declined' = ANY(ARRAY_AGG(ac.status)) THEN 'Unsuccessful'
+                  WHEN 'rejected' = ANY(ARRAY_AGG(ac.status)) THEN 'Unsuccessful'
+                  WHEN 'conditions_not_met' = ANY(ARRAY_AGG(ac.status)) THEN 'Unsuccessful'
+                  WHEN 'offer_withdrawn' = ANY(ARRAY_AGG(ac.status)) THEN 'Unsuccessful'
+                  WHEN 'withdrawn' = ANY(ARRAY_AGG(ac.status)) THEN 'Unsuccessful'
+                END status
+            FROM
+                application_forms f
+            LEFT JOIN
+                application_choices ac ON ac.application_form_id = f.id
+            LEFT JOIN
+                candidates c ON f.candidate_id = c.id
+            WHERE
+                NOT c.hide_in_reporting
+                AND f.submitted_at IS NOT NULL
+                AND f.recruitment_cycle_year = #{RecruitmentCycle.current_year}
+                AND f.region_code IN (
+                  'eastern',
+                  'east_midlands',
+                  'london',
+                  'north_east',
+                  'north_west',
+                  'northern_ireland',
+                  'scotland',
+                  'south_east',
+                  'south_west',
+                  'wales',
+                  'west_midlands',
+                  'yorkshire_and_the_humber',
+                  'european_economic_area',
+                  'rest_of_the_world'
+                )
+                AND (
+                  NOT EXISTS (
+                    SELECT 1
+                    FROM application_forms
+                    AS subsequent_application_forms
+                    WHERE f.id = subsequent_application_forms.previous_application_form_id
+                  )
+                )
+            GROUP BY
+                c.id, ac.id, f.id
+        )
+        SELECT
+            raw_data.sex,
+            raw_data.area,
+            raw_data.age_group,
+            raw_data.status,
+            COUNT(*)
+        FROM
+            raw_data
+        GROUP BY
+            raw_data.sex,
+            raw_data.area,
+            raw_data.age_group,
+            raw_data.status
+      SQL
+    end
+
+    def deferred_offers_counts
+      @deferred_offers_counts ||= ActiveRecord::Base
+        .connection
+        .execute(deferred_offers_query)
+        .to_a
+    end
+
+    def deferred_offers_query
```

Comment:
To reduce duplication could this method be combined with `query` plus a couple of args to tweak it for current/previous year etc.?

Reply:
Yea good point. I think i can just add it as a conditional in the where clause and tweak the status case statement